lib/engine/config.ex

```diff
@@ -176,4 +202,8 @@ defmodule Engine.Config do
   defp schedule_update(pid, time \\ 1_000) do
     Process.send_after(pid, :update, time)
   end
+
+  defp schedule_update_active_headend(pid, time \\ 10_000) do
+    Process.send_after(pid, :update_active_headend, time)
```

Comment:
Do we need to continuously read this? If we expect all writes to be done through this app, then wouldn't it be sufficient to just read it once on startup?